skcosmo/decomposition/pcovr.py

```diff
@@ -290,7 +290,7 @@ def fit(self, X, Y, Yhat=None, W=None):
         self.components_ = self.pxt_.T  # for sklearn compatibility
         return self
 
-    def _fit_feature_space(self, X, Yhat, W=None):
+    def _fit_feature_space(self, X, Y, Yhat, W=None):
```

Comment:
Do we use `W` here? If not, because `_fit_feature_space` is private, we can omit it from the signature.

Reply:
Nope, it's unused. I removed it from the call.